### ISSUE TITLE
About ZK and poly-commitment

### ISSUE THREAD

**Issue by yuzhouwudizuilihai** [TAMUCrypto/virgo-plus#2]
Maybe I don't understand it correctly, does the part of the code that uses the Virgo poly-commitment just commit to the input of the circuit? I don't see anything about the "zk-sumcheck" with mask-polynomials, does the code provide the zk property?

**Comment by dreamATD** [comment]
Thank you for pointing that out! You are right, currently, this version doesn't have zero-knowledge property, it's just an interactive proof protocol (although including ZK won't add too much overhead in the running time). I saw the README might mislead people, thus I'll rephrase it.

**Comment by yuzhouwudizuilihai** [comment]
Thank you for your reply! Is this code used in the paper "Doubly Efficient Interactive Proofs for General Arithmetic Circuits with Linear Prover Time"  to test efficiency? Is there any other ZK version that can be referred to？ @dreamATD

**Comment by dreamATD** [comment]
Yes, this is used in the experiment section of that paper. There hasn't been a zero-knowledge implementation, but it won't introduce much overhead in efficiency.
